terraform/api.tf

```diff
@@ -0,0 +1,77 @@
+resource "aws_lambda_function" "api" {
+  description      = "API for ${local.namespace}"
+  function_name    = local.api_name
+  role             = aws_iam_role.lambda.arn
+  runtime          = "nodejs14.x"
```

Comment:
I know currently we're running a lot of node 14, but it might be worth using a newer version

Reply:
I think 14 is the latest supported by lambda